### PR TITLE
rename frontdoor to frontDoor in readme

### PR DESCRIPTION
**Pull request by hytao**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/frontdoor/resource-manager/readme.nodejs.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 ``` yaml $(nodejs)
 nodejs:
   azure-arm: true
-  package-name: azure-arm-frontdoor
-  output-folder: $(node-sdks-folder)/lib/services/frontdoorManagement
+  package-name: azure-arm-frontDoor
```
Comment (daschult) [inline specification/frontdoor/resource-manager/readme.nodejs.md]: NPM specifies that package names must be all lowercased. Please revert the changes to this file.

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by annatisch** [comment]
Thanks @hytao - do you currently have any published SDKs?
As this PR changes the namespace of the SDK package in each language - which effectively creates a completely new/different release.

Additionally, in some languages the changes shouldn't apply - for example in Python, by convention a package name should be all lower case - please check the conventions in other languages as well.

**Comment by hytao** [comment]
Hi @annatisch , I do not have any published SDKs yet. 

I mainly want to change the namespace from Microsoft.Azure.Management.Frontdoor  to Microsoft.AzureManagement.FrontDoor. for c#.

I'll revert the changes for other languages.

File: specification/frontdoor/resource-manager/readme.nodejs.md (hunk 1 above)
Comment (hytao) [reply specification/frontdoor/resource-manager/readme.nodejs.md]: changes reverted. I think I only need the changes for c#

**Review by ghost — APPROVED** [review]

**Comment by annatisch** [comment]
@AutorestCI regenerate azure-sdk-for-python

**Comment by hytao** [comment]
Hi @annatisch , when will this PR be merged? I'm still waiting for this changes to publish the .net SDK.
Thanks.

**Review by annatisch — APPROVED** [review]
